src/harness/testcases/testdata/esc_sensor_record_0.json

```diff
@@ -0,0 +1,12 @@
+{
+  "id": "esc_sensor/admin_0/esc_sensor_0",
+ "installationParam":{
+   "latitude": 39.0119,
+    "longitude": -98.4842,
+    "height": 9.3,
+    "heightType": "AGL",
+    "indoorDeployment": false,
```

Comment:
R2-ESC-07: ESC Interference Protection:  An ESC operator may request interference protection for one or more of its ESC sensors. An ESC operator that requests protection shall provide to its affiliated SAS(s) the location and height of the protected sensor antenna, as well as the sensor system’s maximum antenna gain in all horizontal directions, specified at 1 degree increments referenced to true north. [Reference R2-SGN-08 and R2-SGN-09].

Based on R2-ESC-07: I think the required fields will be latitude, longitude, height, heightType and antennaGain(which is actually an array with 1-degree increments)